From: Jan Engelhardt <jengelh@inai.de>
Date: 2016-10-17 11:30:09.349081315 +0200

libpff.h #includes <libbfio.h> and therefore, a require tag is needed.
(At least on openSUSE where LIBPFF_HAVE_BFIO is true.)
This probably should be even more conditionalized upon LIBPFF_HAVE_BFIO,
but I leave that to upstream.
---
 libpff.pc.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libpff-20131028/libpff.pc.in
===================================================================
--- libpff-20131028.orig/libpff.pc.in
+++ libpff-20131028/libpff.pc.in
@@ -9,4 +9,4 @@ Version: @VERSION@
 Libs: -L${libdir} -lpff
 Libs.private: @ax_libbfio_pc_libs_private@ @ax_libcdata_pc_libs_private@ @ax_libcerror_pc_libs_private@ @ax_libcfile_pc_libs_private@ @ax_libclocale_pc_libs_private@ @ax_libcnotify_pc_libs_private@ @ax_libcpath_pc_libs_private@ @ax_libcsplit_pc_libs_private@ @ax_libcthreads_pc_libs_private@ @ax_libfcache_pc_libs_private@ @ax_libfdata_pc_libs_private@ @ax_libfdatetime_pc_libs_private@ @ax_libfguid_pc_libs_private@ @ax_libfmapi_pc_libs_private@ @ax_libfwnt_pc_libs_private@ @ax_libuna_pc_libs_private@ @ax_pthread_pc_libs_private@ @ax_zlib_pc_libs_private@
 Cflags: -I${includedir}
-
+Requires: libbfio
